src/controllers/check.your.answers.controller.ts

```diff
@@ -39,8 +39,12 @@ export const get = async (req: Request, res: Response, next: NextFunction) => {
   return next(new Error("No Session present"));
 };
 
-export const post = async (req: Request, res: Response) => {
-  await submitObjection(req.session as Session);
-
-  return res.redirect(OBJECTIONS_CONFIRMATION);
+export const post = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    await submitObjection(req.session as Session);
+    return res.redirect(OBJECTIONS_CONFIRMATION);
+  } catch (e) {
```

Comment:
This stops the web app timing out when there’s an issue submitting the objection. If the REST call fails for any reason the ‘Sorry, there is a problem with the service’ error screen will display promptly. @markpit this changes behaviour for the JS off - double-click scenario but I think it’s required as we then correctly handle all the other potential error scenarios.